scripts/functions/test/testTimeUtils.sh

```diff
@@ -0,0 +1,36 @@
+
+# Copyright 2022 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "$(dirname "${BASH_SOURCE[0]}")/../timeUtils.sh"
+source "$(dirname "${BASH_SOURCE[0]}")/runTestUtils.sh"
+
+expect_non_empty_string "$(time_str)"
+
+START=$(record_time)
+END=$(record_time)
+expect_non_empty_string "$(elapsed_time_str "${START}" "${END}")"
+
```

Comment:
May be worth adding some tests here for `elapsed_time_str` and evaluating the values? (e.g with 10s and 30s, making sure it returns 20 seconds)

Reply:
Good idea, I've added that test.